src/nullablevector.jl

```diff
@@ -311,3 +313,27 @@ function Base.empty!(X::NullableVector)
     empty!(X.isnull)
     return X
 end
+
+function vcat(A::AbstractVector, B::AbstractVector...)
```

Comment:
Similar sentiments here. I'll implement the 2 argument cases for `hcat/vcat` and now at least we have everything written up and documented.